src/OrbitGl/App.cpp

```diff
@@ -352,6 +345,35 @@ void OrbitApp::OnTracepointEvent(orbit_client_protos::TracepointEventInfo tracep
       is_same_pid_as_target);
 }
 
+void OrbitApp::UpdateModulesAbortIfModuleWithoutBuildIdReloader(
+    absl::Span<const ModuleInfo> module_infos) {
+  auto updated_modules = module_manager_->AddOrUpdateModules(module_infos);
+
+  if (!updated_modules.empty()) {
+    std::string module_paths;
+    for (const auto* updated_module : updated_modules) {
+      if (!module_paths.empty()) module_paths += ", ";
+      module_paths += updated_module->file_path();
+      FATAL(
```

Comment:
Display Error to user instead?
And then abort capture? There should be methods for that.

Reply:
I will add a todo here.

Reply:
Since in this case we want to not unload the modules so that after aborting the capture the client is in the consistent state, I will do it in the follow-up change. It ends up being bigger then I expected.